charts/mirrorbits/templates/deployment.yaml

```diff
@@ -25,18 +25,6 @@ spec:
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
       containers:
-        - name: crond
```

Comment:
I moved the order as it's annoying that the default container is crond when you don't specify anything 👅 

like 10 extra characters to exec into the pod.